samples/qa-a11y/src/views/TabbedPanels.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/jsx-no-bind */
+
+import Button from '@enact/agate/Button';
+import CheckboxItem from '@enact/agate/CheckboxItem';
+import {Panel, TabbedPanels} from '@enact/agate/Panels';
+import React from 'react';
+
+const tabTitles = [
+	{title: 'A'},
+	{title: 'B'},
+	{title: 'C'},
+	{title: 'D'}
+];
+
+const TabbedPanelsView = () => {
+	const [noAnimation, setNoAnimation] = React.useState(false);
+	const [noCloseButton, setNoCloseButton] = React.useState(false);
+	const [vertical, setVertical] = React.useState(false);
+
+	const handleNoAnimation = () => setNoAnimation(!noAnimation);
+	const handleNoCloseButton = () => setNoCloseButton(!noCloseButton);
+	const handleVertical = () => setVertical(!vertical);
+
+	return (
+		<TabbedPanels
+			noAnimation={noAnimation}
+			noCloseButton={noCloseButton}
+			orientation={vertical ? 'vertical' : 'horizontal'}
```

Comment:
When `vertical`, it seems linked with the menu panels.

Reply:
It is the GUI issue in qa-a11y. I hope that we could resolve it in a separate issue if we want to fix it.

Reply:
Can we have a left margin from TabbedPanels? to distinguish from the menu selector.